db/db_impl/db_impl.cc

```diff
@@ -3138,6 +3138,7 @@ Status DestroyDB(const std::string& dbname, const Options& options,
   ImmutableDBOptions soptions(SanitizeOptions(dbname, options));
   Env* env = soptions.env;
   std::vector<std::string> filenames;
+  bool wal_not_in_db_path = !IsWalDirSameAsDBPath(&soptions);
```

Comment:
Can we swap the boolean as `wal_in_db_path`? Now a `false` is a double negative and may be hard to understand for some people.